netskope.json

```diff
@@ -10,12 +10,14 @@
     "product_version_regex": ".*",
     "publisher": "Netskope",
     "license": "Copyright (c) 2018-2020 Splunk Inc.",
```

Comment:
copyright year needs to be updated. not sure why the copyright year in static tests test didn't run 🤔